CONTRIBUTING.md

```diff
@@ -455,6 +501,42 @@ Some important commands to know about:
 
 This module can be used to mock module variables, environment variables etc. Check out the official [monkeypatch documentation](https://docs.pytest.org/en/6.2.x/monkeypatch.html).
 
+
+Building Read the Docs (RTD) locally
+====================================
+
+Contributing to the Covalent documentation is highly encouraged. This could mean adding How-To guides, tutorials etc. The steps required to build the RTD locally are listed below:
+
+#### Installation
+
+Navigate to the `/covalent/doc` folder and install the packages to build the documentation:
+
+```bash
+pip install -r requirements.txt
+```
+
+Note that some additional packages might need to be installed as well, depending on the operating system.
+
+#### Building the docs
+
+Navigate to the root folder (`/covalent/`) of the repo and run:
+
+```bash
+python setup.py docs
+```
+
```

Comment:
Add the following section:

Note: You might need to install pandoc via `pip install pandoc`.